crm_stage_type/__manifest__.py

```diff
@@ -1,16 +1,16 @@
 # Copyright 2018 Eficent Business and IT Consulting Services, S.L.
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "CRM Stage Type",
     "summary": "Add type in the lead and opportunity stages",
-    "version": "12.0.1.0.0",
-    "category": "CRM",
+    "version": "13.0.1.0.0",
+    "category": "Customer Relationship Management",
```

Comment:
@skeller1 You need to follow Odoo standard categories in order to appear in OCA shop right category. In CRM, it's 'Sales'.

Reply:
Changed the category to `Sales` and changed tests (using SavepointCase)

**Question: Why all the other OCA/crm addons have a custom category?**

Reply:
> Question: Why all the other OCA/crm addons have a custom category?

Everybody do not pay attention to that :smile:

Reply:
Okay, for further developments: only use these default categories, correct?

https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/data/ir_module_category_data.xml

thank you